core/core_bind.cpp

```diff
@@ -41,39 +41,41 @@
 #include "core/os/keyboard.h"
 #include "core/os/os.h"
 
-////// _ResourceLoader //////
+namespace core_bind {
```

Comment:
Shouldn't it be:

```suggestion
namespace CoreBind {
```

Reply:
I took it from the original PR, and it seems common - if we have CamelCase for namespaces in Godot, I can switch it to that.

Reply:
Namespaces in C++ are generally lower case/snake case. See for instance Google's C++ style guide: https://google.github.io/styleguide/cppguide.html#Namespace_Names.

Reply:
We don't have strong guidelines on this as we haven't been using namespaces much so far. Current `master` branch does have some and most are PascalCase:
```
$ rg -g'!thirdparty' -g'!mono' "^namespace "
tests/test_aabb.h
40:namespace TestAABB {

tests/test_path_follow_2d.h
39:namespace TestPathFollow2D {

tests/test_crypto.h
38:namespace TestCrypto {

tests/test_local_vector.h
38:namespace TestLocalVector {

tests/test_dictionary.h
40:namespace TestDictionary {

tests/test_geometry_2d.h
39:namespace TestGeometry2D {

tests/test_path_3d.h
39:namespace TestPath3D {

tests/test_vector.h
38:namespace TestVector {

tests/test_shader_lang.h
36:namespace TestShaderLang {

tests/test_path_follow_3d.h
39:namespace TestPathFollow3D {

tests/test_hashing_context.h
38:namespace TestHashingContext {

tests/test_physics_3d.h
36:namespace TestPhysics3D {

tests/test_variant.h
39:namespace TestVariant {

tests/test_image.h
40:namespace TestImage {

tests/test_marshalls.h
38:namespace TestMarshalls {

tests/test_physics_3d.cpp
410:namespace TestPhysics3D {

tests/test_text_server.h
40:namespace TestTextServer {

tests/test_translation.h
40:namespace TestTranslation {

tests/test_gui.cpp
59:namespace TestGUI {

tests/test_astar.h
43:namespace TestAStar {

tests/test_pck_packer.h
40:namespace TestPCKPacker {

tests/test_node_path.h
38:namespace TestNodePath {

tests/test_utils.h
36:namespace TestUtils {

tests/test_expression.h
38:namespace TestExpression {

tests/test_math.h
36:namespace TestMath {

tests/test_oa_hash_map.h
36:namespace TestOAHashMap {

tests/test_gui.h
36:namespace TestGUI {

tests/test_geometry_3d.h
41:namespace Test3DGeometry {

tests/test_render.cpp
44:namespace TestRender {

tests/test_resource.h
41:namespace TestResource {

tests/test_xml_parser.h
41:namespace TestXMLParser {

tests/test_basis.h
40:namespace TestBasis {

tests/test_json.h
38:namespace TestJSON {

tests/test_array.h
43:namespace TestArray {

tests/test_command_queue.h
45:namespace TestCommandQueue {

tests/test_rect2.h
38:namespace TestRect2 {

tests/test_file_access.h
37:namespace TestFileAccess {

tests/test_math.cpp
51:namespace TestMath {

tests/test_string.h
45:namespace TestString {

tests/test_gradient.h
40:namespace TestGradient {

tests/test_ordered_hash_map.h
41:namespace TestOrderedHashMap {

tests/test_class_db.h
45:namespace TestClassDB {

tests/test_paged_array.h
38:namespace TestPagedArray {

tests/test_oa_hash_map.cpp
36:namespace TestOAHashMap {

tests/test_color.h
38:namespace TestColor {

tests/test_list.h
38:namespace TestList {

tests/test_object.h
59:namespace TestObject {

tests/test_random_number_generator.h
37:namespace TestRandomNumberGenerator {

tests/test_physics_2d.cpp
398:namespace TestPhysics2D {

tests/test_config_file.h
38:namespace TestConfigFile {

tests/test_curve.h
38:namespace TestCurve {

tests/test_physics_2d.h
36:namespace TestPhysics2D {

tests/test_method_bind.h
42:namespace TestMethodBind {

core/variant/type_info.h
66:namespace GodotTypeInfo {

tests/test_render.h
36:namespace TestRender {

scene/resources/surface_tool.cpp
907:namespace {

tests/test_shader_lang.cpp
44:namespace TestShaderLang {

platform/uwp/app.h
43:namespace GodotUWP

modules/regex/tests/test_regex.h
39:namespace TestRegEx {

modules/navigation/nav_utils.h
44:namespace gd {

modules/fbx/fbx_parser/FBXDocumentUtil.cpp
83:namespace FBXDocParser {
84:namespace Util {

modules/fbx/fbx_parser/FBXDocument.h
50:namespace FBXDocParser {
1234:namespace std {

modules/fbx/fbx_parser/FBXImportSettings.h
79:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXMaterial.cpp
86:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXPose.cpp
81:namespace FBXDocParser {

modules/fbx/fbx_parser/ByteSwapper.h
82:namespace FBXDocParser {
244:namespace Intern {

modules/fbx/fbx_parser/FBXAnimation.cpp
83:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXTokenizer.h
86:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXModel.cpp
82:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXUtil.cpp
82:namespace FBXDocParser {
83:namespace Util {

modules/fbx/fbx_parser/FBXBinaryTokenizer.cpp
84:namespace FBXDocParser {
140:namespace {

modules/fbx/fbx_parser/FBXNodeAttribute.cpp
82:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXDeformer.cpp
85:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXParser.h
90:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXParser.cpp
90:namespace {
107:namespace FBXDocParser {
516:namespace {

modules/fbx/fbx_parser/FBXUtil.h
82:namespace FBXDocParser {
84:namespace Util {

modules/fbx/fbx_parser/FBXTokenizer.cpp
83:namespace FBXDocParser {
101:namespace {

modules/fbx/fbx_parser/FBXCommon.h
80:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXProperties.cpp
82:namespace FBXDocParser {
94:namespace {

modules/fbx/fbx_parser/FBXProperties.h
85:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXMeshGeometry.h
89:namespace FBXDocParser {

modules/mbedtls/tests/test_crypto_mbedtls.h
38:namespace TestCryptoMbedTLS {

modules/fbx/fbx_parser/FBXMeshGeometry.cpp
85:namespace FBXDocParser {

modules/fbx/fbx_parser/FBXDocumentUtil.h
84:namespace FBXDocParser {
85:namespace Util {

modules/mbedtls/tests/test_crypto_mbedtls.cpp
36:namespace TestCryptoMbedTLS {

modules/fbx/fbx_parser/FBXDocument.cpp
90:namespace FBXDocParser {

editor/debugger/debug_adapter/debug_adapter_types.h
37:namespace DAP {

modules/gdscript/tests/test_gdscript.cpp
50:namespace GDScriptTests {

modules/gdscript/tests/gdscript_test_runner.cpp
48:namespace GDScriptTests {

servers/physics_3d/gjk_epa.cpp
86:namespace GjkEpa2 {

modules/gdscript/tests/gdscript_test_runner.h
40:namespace GDScriptTests {

modules/gdscript/tests/test_gdscript.h
37:namespace GDScriptTests {

servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.h
38:namespace RendererSceneRenderImplementation {

servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.h
38:namespace RendererSceneRenderImplementation {

servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h
41:namespace RendererSceneRenderImplementation {

servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.h
40:namespace RendererSceneRenderImplementation {

modules/gdscript/language_server/lsp.hpp
38:namespace lsp {
349:namespace TextDocumentSyncKind {
658:namespace DiagnosticSeverity {
766:namespace MarkupKind {
829:namespace CompletionItemKind {
862:namespace InsertTextFormat {
1076:namespace SymbolKind {
1338:namespace FoldingRangeKind {
1398:namespace CompletionTriggerKind {

drivers/png/png_driver_common.cpp
38:namespace PNGDriverCommon {

drivers/png/png_driver_common.h
36:namespace PNGDriverCommon {

modules/gdnative/tests/test_variant.h
39:namespace TestGDNativeVariant {

modules/gdscript/tests/gdscript_test_runner_suite.h
37:namespace GDScriptTests {

tests/test_lru.h
39:namespace TestLRU {

tests/test_time.h
47:namespace TestTime {
```

PascalCase looks more natural to me in Godot's context, but again since we don't use those much, it's not set in stone.